tests/routes/patch_cancel_booking_by_token_test.py

```diff
@@ -0,0 +1,293 @@
+from datetime import datetime, timedelta
+
+from models import ApiKey, PcObject, EventType, Booking
+from tests.conftest import clean_database, TestClient
+from tests.test_utils import create_stock_with_thing_offer, \
+    create_venue, create_offerer, \
+    create_user, create_booking, create_offer_with_event_product, \
+    create_event_occurrence, create_stock_from_event_occurrence, create_user_offerer, create_stock_with_event_offer, \
+    create_stock, create_deposit
+from utils.token import random_token
+
+
+def create_api_key_for_offerer(offerer, token):
+    offerer_api_key = ApiKey()
+    offerer_api_key.value = token
+    offerer_api_key.offererId = offerer.id
+    return offerer_api_key
+
+
+class Patch:
+    class Returns204:
+        @clean_database
+        def test_should_returns_204_with_cancellation_allowed(self, app):
+            # Given
+            pro_with_user = create_user(public_name='Mr Books', email='Mr Books@example.net')
```

Comment:
Nom de cette variable peu claire et espace dans l'adresse mail
```suggestion
            user = create_user(public_name='Mr Books', email='Mr_Books@example.net')
```

Reply:
dans les autres tests de route  (exemple https://github.com/betagouv/pass-culture-api/pull/856/files#diff-7ab4c139df16439b35f47af3c803e5e6) c'est un pro_user soit un utilisateur pro, ça te paraît plus parlant ?
`pro_user = create_user(email='pro@email.fr')`

Reply:
carrément !